addon/plugins/variable-plugin/variables/person.ts

```diff
@@ -58,16 +78,16 @@ const parseDOM = [
 
 const serialize = (node: PNode, state: EditorState): DOMOutputSpec => {
   const t = getTranslationFunction(state);
-  const mandatee = node.attrs.mandatee as Mandatee;
+  const person = node.attrs.value as Person | undefined;
   return renderRdfaAware({
     renderable: node,
     tag: 'span',
     attrs: {
       ...node.attrs,
```

Comment:
this renders out the bare "value" attribute, with a value of "[object Object]", which is a bit ugly

Reply:
Fixed that.